src/ol_infrastructure/infrastructure/aws/network/Pulumi.infrastructure.aws.network.Dev.yaml

```diff
@@ -1,2 +1,7 @@
 secretsprovider: awskms://alias/PulumiSecrets
 encryptedkey: AQICAHionUR8LBW1ALuVC0rCH3AE2oQIfGMCx3XmpDH9HjM2LQE5JVlRpVbYeiNKFwqxSeLFAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMCnctp6kjfnFHFrOCAgEQgDukpJ8nxbBFRMGqo5VuudSbWHeCV5XzBVj28cgetVPMBOlbke4WOClmQi4cYkKYfAKWgSvDmQUtAb7rUg==
+config:
+  aws:region: us-east-1
+  data_vpc:cidr_block: 10.200.0.0/16
+  operations_vpc:cidr_block: 10.201.0.0/16
```

Comment:
Should there be a `data_vpc:name`?

Reply:
I added the operations VPC name because of the fact that it already exists and we need to import it.